containers/ws/label-run

```diff
@@ -42,8 +42,10 @@ else
 
     /usr/libexec/cockpit-certificate-ensure
 
-    if [ -z "$SSH_AUTH_SOCK" ]; then
+    if [ -z ${SSH_AUTH_SOCK+x} ]; then
```

Comment:
Please write that in standard shell syntax and with quotes: `[ -z "${SSH_AUTH_SOCK:-}" ]`